### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by KenbonTN**
- [ ] There is no linter error.
- [ ] I have added descriptive README file
- [ ] Learned how to parse a Figma design to create a UI.
- [ ] In order to lay out the elements on the page I used Flex-box in all 2 sections.
- [ ] Functionalities that requires JavaScript are not implemented, like opening the menu or creating drop-downs.

### CODE REVIEW

File: styles/mobile-first-style.css [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+.font-style {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+}
+
+.main {
+  background-image: url(/images/header-shapes\ mobile@2x.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  margin-left: 36px;
```
Comment (Felix45) [inline styles/mobile-first-style.css]: - [ Required ] Please consider removing the `margin-left: 36px` CSS rule from the `.main` class. This margin is not consistent with the `margins` on the Figma design, it's good practice to always try as much as possible to be consistent with the Figma design. Please see the screenshot below for more details on this.

## Figma template


## Your implementation

File: index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    
+                                <li>
+                                    <a class="social_medias" href="https://www.linkedin.com/in/kenbon-teshome/">
+                                        <svg xmlns="http://www.w3.org/2000/svg" width="23" height="23" fill="currentColor"
+                                            class="bi bi-linkedin icon" viewBox="0 0 16 16">
+                                            <path
+                                                d="M0 1.146C0 .513.526 0 1.175 0h13.65C15.474 0 16 .513 16 1.146v13.708c0 .633-.526 1.146-1.175 1.146H1.175C.526 16 0 15.487 0 14.854V1.146zm4.943 12.248V6.169H2.542v7.225h2.401zm-1.2-8.212c.837 0 1.358-.554 1.358-1.248-.015-.709-.52-1.248-1.342-1.248-.822 0-1.359.54-1.359 1.248 0 .694.521 1.248 1.327 1.248h.016zm4.908 8.212V9.359c0-.216.016-.432.08-.586.173-.431.568-.878 1.232-.878.869 0 1.216.662 1.216 1.634v3.865h2.401V9.25c0-2.22-1.184-3.252-2.764-3.252-1.274 0-1.845.7-2.165 1.193v.025h-.016a5.54 5.54 0 0 1 .016-.025V6.169h-2.4c.03.678 0 7.225 0 7.225h2.4z" />
+                                        </svg>
+                                    </a>
+                                </li>
```
Comment (Felix45) [inline index.html]: - [ Required ] Please consider changing the `LinkedIn` Icon to match the background-color on the Figma template, In the  Figma template all the `icons` have the same background-color, whereas on your design the `LinkedIn` icon has a different color. Please see the screenshot below for more details on this. 
- [ Required ] Please make sure your `padding` and `margins` are consistent with the `Figma` template.

## Figma template
Screenshot from 2023-01-12 15-05-22

## Your Implementation
Screenshot from 2023-01-12 14-59-58

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @KenbonTN , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :heavy_check_mark: 
- [ ] All linter checks are setup correctly and passing with green checks :heavy_check_mark: 
- [ ] Professional readme file :heavy_check_mark: 
- [ ] GitHub flow is used correctly :heavy_check_mark: 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @KenbonTN 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors present
- The PR has a proper title and a descriptive body 👍

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**